packages/std-client/src/setup/setupMUDV2Network.ts

```diff
@@ -143,8 +144,7 @@ export async function setupMUDV2Network<C extends ContractComponents, S extends
   const singletonEntity = world.registerEntity({ id: SingletonID });
   // Register player entity
   const address = network.connectedAddress.get();
-  const playerEntityId = address;
-  const playerEntity = playerEntityId ? world.registerEntity({ id: playerEntityId }) : undefined;
+  const playerEntity = address ? world.registerEntity({ id: keyTupleToEntityID([address]) }) : undefined;
```

Comment:
this was still registering the unpadded version, which means if you used this anywhere in a v2 app, you'd get the wrong results for a lookup/query